indico/modules/categories/templates/management/content.html

```diff
@@ -74,8 +76,16 @@
 {% endblock %}
 
 {% block content %}
-    {{ subcategories_table(subcategories) }}
-    <script>
-        setupCategoryTable();
-    </script>
+    {% if subcategories %}
```

Comment:
`or not event_count`? I think we said we want to show both if theres nothing in the category at all